src/App.vue

```diff
@@ -26,7 +26,7 @@
 
             <stepper class="mt-10" />
 
-            <router-view />
+            <router-view :key="isDraft"/>
```

Comment:
Is this to cause the router-view page to redisplay?

Interesting... This could be used to re-route to the current page (which is currently not possible in Vue -- or Angular and probably others). (For example, we might re-route to the dashboard when the user clicks the header logo, forcing the page to refresh all data. Just a thought.)

Reply:
Exactly!  I did a little research on how people would 'forceUpdates()' of the UI. Though it was widely suggested you shouldn't want/have to do that. 

What was recommended was using a key on the components you would want to re-render. When the key changes the components flagged will update.

 I found that none of the traditional component lifecycle hooks would time properly because the components would mount before the data was retrieved ( if there was any draft data ).  I'm sure we could still expand and improve on this approach, and i'm 100% open to any suggestions!